recommenders/models/sar/sar_singlenode.py

```diff
@@ -565,3 +566,17 @@ def predict(self, test):
             }
         )
         return df
+
+
+if __name__ == '__main__':
```

Comment:
This piece of code seems to have been checked in by mistake.

Reply:
oops! fixed.